### PR TITLE
Disable channel mode switches when user is not an op

### LINKED ISSUE #176: Channel modes + settings readonly when no chan op modes

### PR DESCRIPTION
**Pull request by ItsOnlyBinary**
closes #176

@AllanOcelot this could do with some styling so the user can easily see they are disabled

### CODE REVIEW

**Comment by ItsOnlyBinary** [comment]
^^^ rebased

**Comment by prawnsalad** [comment]
Making all the content 50% transparent to show that they're disabled is just making it difficult to see here. How about instead of the thick outlines that the editable inputs have, they are removed so they're  more like labels instead of editable areas?

**Comment by AllanOcelot** [comment]
@prawnsalad  and @ItsOnlyBinary  this could do with another look  - I've tried instead to just put a notice at the top of the channel settings, simple and effective :)

**Comment by ItsOnlyBinary** [comment]
Don't think i like that either :( what about just hiding the options all together if they are not op.

that way at least it would make the `Your settings` more visible

File: src/components/Sidebar.vue [hunk 1]
```diff
@@ -27,10 +27,15 @@
 
                     <tabbed-view>
                         <tabbed-tab :header="$t('settings')" :focus="true">
-                            <h3>{{ $t('channel_settings') }}</h3>
-                            <hr>
-                            <channel-info :buffer="buffer"/>
-
+                          <h3>{{ $t('channel_settings') }}
+                              <span v-if="!areWeAnOp()"
+                                    class="kiwi-channelsettings-disabled">
```
Comment (ItsOnlyBinary) [inline src/components/Sidebar.vue]: newline the `if` and `>`

**Comment by AllanOcelot** [comment]
@ItsOnlyBinary  @prawnsalad  did we come to a conclusion on this PR?

**Comment by ItsOnlyBinary** [comment]
rebased

**Pull request closed without merging**